testdata/project-v3-with-deploy-image/config/rbac/role.yaml

```diff
@@ -51,6 +51,13 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
@NikhilSharmaWe 

I added the review in : https://github.com/NikhilSharmaWe/kubebuilder/pull/41/files

See that the RBAC needs to be fixed also is better if we use Warning in the example so that we can easily check in the tests as well.